packages/spaces/src/lib/SpacesDisclaimer.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { render, waitFor } from '@testing-library/react';
+// eslint-disable-next-line @nx/enforce-module-boundaries
+import { server } from '../../../mock/src/lib/server';
```

Comment:
I believe you should be able to reference this with `@availity/mock` since it is in the `tsconfig.base.json`.

Reply:
I have not been able to get that to work in the past, but I'll take another swing at it.

Reply:
fixed